README.md

```diff
@@ -15,7 +15,7 @@ Flight Review is deployed on https://review.px4.io.
 
 - clone the repository
 - use python3
-- `pip3 install bokeh jinja2 pyulog simplekml` (at least version 0.12.6
+- `pip3 install bokeh jinja2 pyulog simplekml pandas scipy` (at least version 0.12.6
```

Comment:
I think `pandas` can be removed from here and the travis.yml file (It was required at some point, but not anymore)